model_prices_and_context_window.json

```diff
@@ -1,4 +1,16 @@
 {
+    "ERNIE-bot":{
+        "max_tokens": 4096,
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096,
+        "input_cost_per_token": 0.000005,
+        "output_cost_per_token": 0.000015,
+        "litellm_provider": "Baidu",
```

Comment:
i'm confused. this isn't even a supported litellm provider - https://docs.litellm.ai/docs/providers

what is this for?